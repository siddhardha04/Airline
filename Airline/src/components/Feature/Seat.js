import React from "react";
import PropTypes from "prop-types";

export default function Seat(props) {
  let color = props.color;
  let seatNo = props.seatNo;

  let seatColor = {
    backgroundColor: color
  };

  return (
    <div className="seat" style={seatColor}>
      <h6 className="seat-num">{seatNo}</h6>
    </div>
  );
}

Seat.propTypes = {
  color: PropTypes.string.isRequired,
  seatNo: PropTypes.string.isRequired
};
