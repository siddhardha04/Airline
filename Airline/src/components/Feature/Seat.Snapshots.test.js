import React from "react";
import Seat from "./Seat";
import renderer from "react-test-renderer";

it("Seat Will have B5 as seat no", () => {
  const tree = renderer.create(<Seat color={"green"} seatNo="B5" />);
  expect(tree).toMatchSnapshot();
});
