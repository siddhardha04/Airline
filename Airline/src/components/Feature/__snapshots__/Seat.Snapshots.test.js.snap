// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seat Will have B5 as seat no 1`] = `
<div
  className="seat"
  style={
    Object {
      "backgroundColor": "green",
    }
  }
>
  <h6
    className="seat-num"
  >
    B5
  </h6>
</div>
`;
